sysutils/diffutils-for-muniversal/Portfile

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                diffutils-for-muniversal
+version             3.8
+revision            0
+platforms           {darwin > 21}
+
+# we will not use the use_xz mechanism, as we don't want to pull in MP xz port and deps
+distname            diffutils-${version}
+extract.suffix      .tar.xz
+
+categories          sysutils
+license             GPL-3+
+
+maintainers         nomaintainer
+
+description         GNU diff utilities sequestered for muniversal PG
+long_description \
+    diffutils contains the GNU diff, diff3, sdiff, and cmp utilities. \
+    Sequestered for muniversal to use.
+
+homepage            https://www.gnu.org/software/diffutils/
+
+# this looks in wrong subdir due to name change.. how to fix?
+#master_sites        gnu
+
+master_sites        https://www.mirrorservice.org/sites/ftp.gnu.org/gnu/diffutils/
+
+checksums           rmd160  30b1050d0a5334becdc6141dcd52a4b584f0dbc4 \
+                    sha256  a6bdd7d1b31266d11c4f4de6c1b748d4607ab0231af5188fc2533d0ae2438fec \
+                    size    1585120
+
+# boilerplate extract phase override
+extract {
+    foreach f ${distfiles} {
+        set f [lindex [split ${f} :] 0]
+        ui_debug "Extracting: ${f}"
+        if {[string match "*bz2*" ${f}]} {
+            system -W ${workpath} "/usr/bin/tar xjf ${distpath}/${f}"
+        }
+        if {[string match "*gz*" ${f}]}  {
+            system -W ${workpath} "/usr/bin/tar xzf ${distpath}/${f}"
+        }
+        if {[string match "*xz*" ${f}]}  {
+            system -W ${workpath} "/usr/bin/tar xJf ${distpath}/${f}"
+        }
+    }
+}
+
+# sterilize MacPorts build environment; we want nothing picked up from MP prefix
```

Comment:
Maybe extract it to PG?

Reply:
perhaps, at some future point. although there are so few ports that use this, maybe best to just keep it like this...